docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,472 @@
+    "```\n",
+    "\n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d552ab9",
+   "metadata": {},
+   "source": [
+    "## 1. Install required dependencies and download data\n",
+    "You can use `pip` to install all packages required for this tutorial as follows\n",
+    "```\n",
+    "!pip install cleanlab\n",
+    "!pip insall matplotlib\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0ba0dc70",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "dependencies = [\"cleanlab\", \"matplotlib\"]\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = ' '.join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c90449c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ObjectDetectionBenchmarking/tutorial_obj/predictions.pkl'\n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ObjectDetectionBenchmarking/tutorial_obj/labels.pkl'\n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/ObjectDetectionBenchmarking/tutorial_obj/example_images.zip' && unzip -q -o example_images.zip"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df8be4c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pickle\n",
+    "\n",
+    "from cleanlab.object_detection.rank import get_label_quality_scores, issues_from_scores\n",
+    "from cleanlab.object_detection.filter import find_label_issues\n",
+    "from cleanlab.object_detection.summary import visualize"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2506badc",
+   "metadata": {},
+   "source": [
+    "## 2. Format data, labels and model predictions\n",
+    "\n",
+    "We begin by loading in the `labels`and `predictions` required. For the`predictions` object, we separately trained an object detection model and used it to obtain predictions on a validation dataset in a training notebook [here](https://github.com/cleanlab/examples/blob/master/active_learning_multiannotator/active_learning.ipynb) while the `labels` come from our subset of COCO."
```

Comment:
I believe the training notebook link here is incorrect (links to active learning notebook)